js/binary/reader_test.js

```diff
@@ -42,6 +42,8 @@
  */
 
 goog.require('goog.testing.asserts');
+
+// CommonJS-LoadFromFile: google_protobuf
```

Comment:
`googleProtobuf`... I ran the tests with this file as-is, and everything passed... how is that possible?